generate-renode-scripts.py

```diff
@@ -231,7 +231,7 @@ def generate_spiflash(peripheral, shadow_base, **kwargs):
         string: repl definition of the peripheral
     """
 
-    flash_size = 0x2000000
+    flash_size = 0x10000000
```

Comment:
Flash size should come from the platform?

Reply:
I noticed that not only `flash_size`, but also xip base address were hardcoded in the script. I fixed both by using values from the `csv` file.